ush/generate_FV3LAM_wflow.py

```diff
@@ -438,78 +428,88 @@ def generate_FV3LAM_wflow(
     settings["nam_sfcperts"] = nam_sfcperts_dict
 
     #
-    #-----------------------------------------------------------------------
+    # -----------------------------------------------------------------------
     #
     # Generate namelist files with stochastic physics if needed
     #
-    #-----------------------------------------------------------------------
+    # -----------------------------------------------------------------------
     #
     if any((DO_SPP, DO_SPPT, DO_SHUM, DO_SKEB, DO_LSM_SPP)):
         realize(
-            input_config=FV3_NML_FP,
+            input_config=fv3_nml_fp,
```

Comment:
The generation of the grid_SUBCONUS_Ind_3km_ics_HRRR_lbcs_HRRR_suite_HRRR WE2E test experiment failed because fv3_nml_fp doesn't exist.  The experiment was successfully generated when I changed fv3_nml_fp back to FV3_NML_FP.

```suggestion
            input_config=FV3_NML_FP,
```